_conferences/flutter-friends-september-2025.md

```diff
@@ -10,5 +10,5 @@ date_end:   2025-09-02
 cfp:
   start: 2025-02-01
   end:   2025-07-01
-  site: https://airtable.com/appuBqVNvGwxZ17Hb/shr1ZCFLlvHGH0sTN
+  site:  https://airtable.com/appnt9GgdfcnH4XjT/pagdanm13HR3z3f1k/form
```

Comment:
Extra space is intentional?

Reply:
Yeah, that way it's aligned with the dates in the lines above